.pipelines/nightly.yaml

```diff
@@ -135,12 +135,14 @@ jobs:
         value: "true"
     strategy:
       matrix:
-        kind_v1_24_7:
-          KIND_NODE_VERSION: v1.24.7
-        kind_v1_25_3:
-          KIND_NODE_VERSION: v1.25.3
-        kind_v1_26_0:
-          KIND_NODE_VERSION: v1.26.0
+        kind_v1_24_12:
+          KIND_NODE_VERSION: v1.24.12
+        kind_v1_25_8:
+          KIND_NODE_VERSION: v1.25.8
+        kind_v1_26_3:
+          KIND_NODE_VERSION: v1.26.3
+        kind_v1_27_0:
+          KIND_NODE_VERSION: v1.27.0
```

Comment:
```suggestion
        kind_v1_27_1:
          KIND_NODE_VERSION: v1.27.1
```

Reply:
done! updated the version in the PR yaml as well.